src/Core/Models/RegisterSystemResponse.cs

```diff
@@ -14,15 +14,30 @@ namespace Altinn.Platform.Authentication.Core.SystemRegister.Models
     public class RegisterSystemResponse
     {
         /// <summary>
-        /// The unique Id for this product, in human-readable string format.
-        /// The id is in the format of system_vendor_name_plus_name_chosen_by_them.
+        /// The primary key to store and edit the Registered System.
+        /// Used as a Foreign Key in the System User table.
```

Comment:
I have changed that as well in my new PR :)

Reply:
updated from your branch